### PR TITLE
Final App 

### PR DESCRIPTION
**Pull request by mutinhiri**
In this Pr we built a full booking App @Emmy-github-webdev and @mutinhiri we were working in a group of 2 people 

the link to our frontend pr: https://github.com/mutinhiri/book-an-appointment-frontend/pull/11

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    namespace :v1 do
+      get '/doctors', to: 'doctors#index'
+      get '/doctors/:id', to: 'doctors#show'
+      post '/doctors/create', to: 'doctors#create'
+      post '/users', to: 'users#create'
+      get 'signup', to: 'users#new'
+      get 'login', to: 'sessions#new'
+      post 'login', to: 'sessions#create'
+      delete 'logout', to: 'sessions#destroy'
+      resources :appointments
```
Comment (MuriloRoque) [inline config/routes.rb:9]: - [ ] I advise you to specify the routes you will need here, so tou don't create useless ones.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
Hi @mutinhiri, @Emmy-github-webdev,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

- [ ] Build tests for your app, it's important to make sure your app is working properly.
- [x] Solve your linter errors.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: config/routes.rb (hunk 1 above)
Comment (Emmy-github-webdev) [reply config/routes.rb:9]: Thank you @MuriloRoque 

The specified routes are now removed

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :appointments
+  # before_save { self.email = email.downcase }
+  validates :name, presence: true, uniqueness: { case_sensitive: false }, length: { minimum: 3, maximum: 100 }
+  # VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  # validates :email, uniqueness: { case_sensitive: false },
+  #                  format: { with: VALID_EMAIL_REGEX }
+  # has_secure_password
```
Comment (christianotieno) [inline app/models/user.rb]: Do we have any reason why we don't need this logic anymore? 🤔

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hi @mutinhiri & @Emmy-github-webdev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 2 above)
Comment (Emmy-github-webdev) [reply app/models/user.rb]: @christianotieno; We do not need the logic because full authentication is not implemented.

**Review by MuriloRoque — APPROVED** [review]
Hi @mutinhiri, @Emmy-github-webdev,

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**